etc/scipipe/build_matrix.yaml

```diff
@@ -48,6 +55,9 @@ scipipe-lsstsw-matrix:
     # allow builds on monterey and ventura
     label: osx-12||osx-13
     display_name: macos
+  - <<: *macarm64-conda
+    label: mini
```

Comment:
Since the `label` and `display_name` are in the definition above, you shouldn't need it here.  They're only in the previous one to allow jobs to run on either platform and have the same display name.